### PR TITLE
Add configurations for native-image build

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3451

elated to: https://github.com/ballerina-platform/ballerina-standard-library/issues/3315

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    "name": "com.mysql.cj.protocol.StandardSocketFactory",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "name": "io.netty.bootstrap.ServerBootstrap$1"
```
Comment (TharmiganK) [inline native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json:192]: Do we need this netty class? Can we check by removing this?

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+        "name": "setNetworkTimeout",
+        "parameterTypes": [
+          "java.util.concurrent.Executor",
+          "int"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "org.quartz.simpl.CascadingClassLoadHelper",
```
Comment (TharmiganK) [inline native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json:233]: quartz related configs are already added through task module. So can we remove this and check?
Also check other quartz related classes

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/resource-config.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "\\Qcom/mysql/cj/TlsSettings.properties\\E"
+      },
+      {
+        "pattern": "\\Qorg/quartz/core/quartz-build.properties\\E"
```
Comment (TharmiganK) [inline native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/resource-config.json:8]: Can we check removing this quartz resource?

Comment (kaneeldias) [reply native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/resource-config.json]: Removed in [9f68040](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/653/commits/9f68040d7ddf069c53730192d708bfa74bc74e81)

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json (hunk 2 above)
Comment (kaneeldias) [reply native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json]: Removed in [9f68040](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/653/commits/9f68040d7ddf069c53730192d708bfa74bc74e81)

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json (hunk 1 above)
Comment (kaneeldias) [reply native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json]: Removed in [9f68040](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/653/commits/9f68040d7ddf069c53730192d708bfa74bc74e81)

**Review by TharmiganK — APPROVED** [review]

**Pull request merged**